roles/setup_lvm/tasks/main.yml

```diff
@@ -4,16 +4,21 @@
       - 'item.value.drives|length > 0'
   with_dict: '{{ metal_k8s_lvm.vgs }}'
 
+- name: "LVM Setup: Check LVM packages"
+  package:
+    name: lvm2
+    state: present
+
 - name: start lvm storage setup
   debug:
-    msg: "Start LVM storage setup"
+    msg: "LVM Setup: Start LVM storage setup"
```

Comment:
Is this tasks needed ? The name os the task is printed as well as the debug output: You're displaying basically (n+1)  times "start lvm storage setup" (one for the name of the task, one for each server you run on).

Reply:
Oh ok, removing it then